django/db/backends/base/base.py

```diff
@@ -723,28 +728,49 @@ def schema_editor(self, *args, **kwargs):
             )
         return self.SchemaEditorClass(self, *args, **kwargs)
 
-    def on_commit(self, func):
+    def on_commit(self, func, is_robust):
```

Comment:
```suggestion
    def on_commit(self, func, is_robust=False):
```
Add default value for better backward compatibility.

Reply:
I didn't add a default value here since the function would receive a value from the `django/db/transaction.py`  on_commit function and a default value is added there. I will make the change if you deem it necessary in base.py file too. 👍

Reply:
`on_commit` can be used directly by 3rd-party database backends, that's why we should add a default value to `is_robust.`